tools/firmware-utils/src/ptgen.c

```diff
@@ -250,6 +507,12 @@ int main (int argc, char **argv)
 		case 'S':
 			signature = strtoul(optarg, NULL, 0);
 			break;
+		case 'G':
+			if (guid_parse(optarg, &guid)) {
+				fprintf(stderr, "Invalid guid string\n");
```

Comment:
`fputs()`

Reply:
The reason for `fputs()` is that `-Wformat` will complain about calls to `printf`-like functions without any format specifiers.